common/lib/symbiosis/ssl/hooks.rb

```diff
@@ -31,18 +31,17 @@ def valid_hook?(hook)
 
       private
 
-      def runs_successfully?(hook)
-        opts = { stdin_data: @domains.join("\n") }
-        output, status = Open3.capture2e([hook, event], opts)
-
+      def runs_successfully?(script)
+        opts = { stdin_data: @domains.join("\n") + "\n" }
+        output, status = Open3.capture2e(script, @event, opts)
         return true if status.success?
-        puts hook_status(output, status)
+        puts script_status(script, output, status)
         false
       end
 
-      def hook_status(output, status)
+      def script_status(script, output, status)
         "============================================\n" \
-        "Error executing SSL script for #{@event} hook\n" \
+        "Error executing SSL script for #{@event} event\n" \
         "#{script} exited with status #{status.exitstatus}\n" \
```

Comment:
Yeah, I also noticed this when @User4574 came over to have a chat a while ago - was gonna push a commit about it myself but you've already done it for me!

Sorry for dropping a few balls with my ruby! Still a bit rusty, it would seem!